geoportal/c2cgeoportal_geoportal/views/entry.py

```diff
@@ -187,14 +200,21 @@ def _get_metadatas(self, item, errors):
         return metadatas
 
     def _wms_getcap(self, ogc_server):
+        loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(loop)
+        result = loop.run_until_complete(self._async_wms_getcap(ogc_server))
+        loop.close()
+        return result
```

Comment:
Why not asyncio.run, python doc says:
Application developers should typically use the high-level asyncio functions, such as asyncio.run(), and should rarely need to reference the loop object or call its methods.

Reply:
Suggested change: 
```
async def _wms_getcap(self, ogc_server):
    ...
    return wms

def _wms_layers(self, ogc_server):
    wms = asyncio.run(self._wms_getcap())
```

Reply:
+1 :-)

Reply:
but not in _wms_layers...